packages/contracts/utils/environment.ts

```diff
@@ -0,0 +1,58 @@
+import {HARDHAT_NETWORK_NAME} from 'hardhat/plugins';
+import {Network} from 'hardhat/types';
+
+/**
+ * Provides default values for environment variables if running against a hardhat node
+ * else requires that the environment variables are set
+ * @param network the hardhat network object
+ * @param name of the environment variable
+ * @param defaultValue the fallback value to be used if not set and in development mode
+ * @returns the value of the environment variable, or a fallback if possible
+ */
+export function env(
+  network: Network,
+  name: string,
+  defaultValue: string
+): string {
+  const value = process.env[name];
+  const isLocal =
+    network.name === HARDHAT_NETWORK_NAME || network.name === 'localhost';
+  if (!isLocal && !value) {
+    throw new Error(`Missing env var: ${name}`);
+  }
+  return process.env[name] || defaultValue;
+}
+
+/// Specific environment variables with hardcoded defaults
+
+export const daoDomainEnv = (network: Network): string =>
+  env(network, `${network.name.toUpperCase()}_DAO_ENS_DOMAIN`, 'dao.eth');
```

Comment:
On sepolia, the domain is `aragon-dao.eth` (because `dao.eth` was already taken on the official ENS registry deployed there).